tests/utils/get_container_health.py

```diff
@@ -4,7 +4,6 @@
 from docker.models.containers import Container
 
 
-def get_health(container: Container) -> str:
-    api_client = docker.APIClient()
```

Comment:
This is funny coincidence.
I refactored code around health check earlier today 😃 

Didn't notice this bug.
We're using separate API client here, not the one from the `docker_client` fixture.

That's why it was impossible to make both healthcheck and ipv6 tests work.
Passing client as param and using it should fix the problem